### PR TITLE
add headings to some ml examples

### PR DESCRIPTION
**Pull request by kning**
Starting with the `06_gpu_and_ml` folder. The `spam_detect` and `text-to-pokemon` apps I'm not entirely sure what to do with

### CODE REVIEW

File: 06_gpu_and_ml/stable_diffusion/flux.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # ---
 # output-directory: "/tmp/flux"
 # ---
-# example originally contributed by [@Arro](https://github.com/Arro)
+# # Flux.1 (Schnell)
+#
+# This example runs the popular [Flux.1-schnell](https://www.google.com/search?client=firefox-b-1-d&q=flux+schnell+) text-to-image model on Modal.
```
Comment (charlesfrye) [inline 06_gpu_and_ml/stable_diffusion/flux.py:6]: This link looks wrong

**Review by charlesfrye — CHANGES_REQUESTED** [review]

**Comment by charlesfrye** [comment]
You can ignore those two examples.

File: 06_gpu_and_ml/stable_diffusion/flux.py (hunk 1 above)
Comment (kning) [reply 06_gpu_and_ml/stable_diffusion/flux.py:6]: 😓

File: 06_gpu_and_ml/stable_diffusion/flux.py [hunk 2]
```diff
@@ -1,7 +1,11 @@
 # ---
 # output-directory: "/tmp/flux"
 # ---
-# example originally contributed by [@Arro](https://github.com/Arro)
+# # Flux.1 (Schnell)
```
Comment (charlesfrye) [inline 06_gpu_and_ml/stable_diffusion/flux.py:4]:
```suggestion
# # Run Flux.1 (Schnell) on Modal
```

File: 06_gpu_and_ml/embeddings/instructor.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# # Instructor Embedding
```
Comment (charlesfrye) [inline 06_gpu_and_ml/embeddings/instructor.py:1]:
```suggestion
# # Create Instructor Embeddings on Modal
```

File: 06_gpu_and_ml/embeddings/text_embeddings_inference.py [hunk 4]
```diff
@@ -1,6 +1,10 @@
 # ---
 # cmd: ["modal", "run", "06_gpu_and_ml/embeddings/text_embeddings_inference.py::embed_dataset"]
 # ---
+# # Text Embeddings Inference
```
Comment (charlesfrye) [inline 06_gpu_and_ml/embeddings/text_embeddings_inference.py:4]:
```suggestion
# # Run TextEmbeddingsInference (TEI) on Modal
```

File: 06_gpu_and_ml/stable_diffusion/playground.py [hunk 5]
```diff
@@ -3,7 +3,10 @@
 # output-directory: "/tmp/playground-2-5"
 # args: ["--prompt", "A cinematic shot of a baby raccoon wearing an intricate Italian priest robe."]
 # ---
-
+# # Playground v2.5
```
Comment (charlesfrye) [inline 06_gpu_and_ml/stable_diffusion/playground.py:6]:
```suggestion
# # Run Playground v2.5 on Modal
```

File: 06_gpu_and_ml/stable_diffusion/stable_diffusion_xl_lightning.py [hunk 6]
```diff
@@ -1,3 +1,11 @@
+# ---
+# output-directory: "/tmp/stable-diffusion-xl-lightning"
+# ---
+# # SDXL Lightning
```
Comment (charlesfrye) [inline 06_gpu_and_ml/stable_diffusion/stable_diffusion_xl_lightning.py:4]:
```suggestion
# # Run SDXL Lightning on Modal
```

File: 06_gpu_and_ml/stable_diffusion/stable_video_diffusion.py [hunk 7]
```diff
@@ -1,6 +1,10 @@
 # ---
 # cmd: ["modal", "serve", "06_gpu_and_ml/stable_diffusion/stable_video_diffusion.py"]
 # ---
+# # Stable Video Diffusion
```
Comment (charlesfrye) [inline 06_gpu_and_ml/stable_diffusion/stable_video_diffusion.py:4]:
```suggestion
# # Run Stable Video Diffusion in a Streamlit app
```

**Review by charlesfrye — APPROVED** [review]

**Pull request merged**